src/dbnode/namespace/options.go

```diff
@@ -75,6 +75,7 @@ type options struct {
 	runtimeOpts           RuntimeOptions
 	extendedOpts          ExtendedOptions
 	aggregationOpts       AggregationOptions
+	stagingState          StagingState
```

Comment:
Should this be added to verify? Probably the only failing case is if this is an unknown value